messages/en.json

```
@@ -1,4 +1,8 @@
 {
-  "admin/test": "test",
-  "store/test": "test"
+  "store/addtocart.success": "Item added to your cart",
```

I think that would be better if we use kebab case here, like:
```suggestion
  "store/add-to-cart.success": "Item added to your cart",
```